src/connectors/snowflake/trulens/connectors/snowflake/connector.py

```diff
@@ -275,6 +293,20 @@ def _set_up_database_args(
         )
         return database_args
 
+    def _set_up_sis_dashboard(
+        self,
+        session: Optional[Session] = None,
+        warehouse: Optional[str] = None,
+        init_server_side_with_staged_packages: bool = False,
+    ) -> None:
+        return SiSDashboardArtifacts(
+            session or self.snowpark_session,
+            self.connection_parameters["database"],
+            self.connection_parameters["schema"],
+            warehouse or self.connection_parameters["warehouse"],
+            init_server_side_with_staged_packages or self.use_staged_packages,
```

Comment:
on line 87 didn't you already set `self.use_staged_packages: bool = init_server_side_with_staged_packages`? 

is the attribute `use_staged_packages` expected to be set in some other places?

Reply:
nope line 87 is the only place we set that variable and its only used here atm

Reply:
oh wait maybe I'm missing sth, but why not just `self.use_staged_packages` when passing to constructing `SiSDashboardArtifacts` here?